packages/contracts/deployments/README.md

```diff
@@ -1,6 +1,15 @@
 # Optimism Regenesis Deployments
 ## LAYER 2
 
+## OPTIMISTIC-KOVAN
+
+Network : __undefined (chain id: 69)__
+
+|Contract|Address|
+|--|--|
+|OVM_GasPriceOracle|[0x668E5b997b9aE88a56cd40409119d4Db9e6d752E](https://undefined.etherscan.io/address/0x668E5b997b9aE88a56cd40409119d4Db9e6d752E)|
```

Comment:
Why is this undefined? Does our code generation script have an error?

Reply:
This is undefined because this was deployed to L2. The script needs to be updated to point to the optimistic etherscan but I don't believe there is a kovan optimistic etherscan

Reply:
https://kovan-optimistic.etherscan.io